app/build.gradle

```diff
@@ -44,6 +44,17 @@ dependencies {
     implementation(dependenciesList.mapboxMaps) {
         exclude group: 'com.mapbox.mapboxsdk', module: 'mapbox-android-gestures'
     }
+    implementation("androidx.core:core-ktx") {
+        version {
+            strictly("1.6.0")
```

Comment:
Why do we need to use strictly here?

Reply:
This is a temporary solution to be able to compile the app with compileSDK = 30 until we publish a Maps SDK that supports it

Reply:
And this is an app module so it doesn't matter for the sdk itself